netdb.h

```diff
@@ -0,0 +1 @@
+#pragma once
```

Comment:
We do not provide any functionality here, so isn't it better to just ifdef all inclusions of this file on Windows?

Reply:
I prefer to make as few changes as possible in controller. Adding a file here is so much easier for us (and me, because of the future merging)